### DISCUSSION TITLE
Crater not generating PDF invoices

### DISCUSSION THREAD

**Discussion by linknixon** [crater-invoice-inc/crater#1215] [Q&A]
I have installed the software v6.0.6 on Ubuntu 20.04 LTS but I have the following issues:

1. I can not set the currency to my country currency UGX because it is not listed.
2. The systems is throwing error 505 on generating PDF invoices.
3. The Invoice Templates can not be customized.
Is there any help I can get on the same? Thank you.

**Comment by linknixon** [comment]
Is there away to get the invoice templates?

**Reply by Yramgar** [reply]
The templates can be customized. Here you will find the templates
/youinstallation/resources/views/app/pdf/estimate
They are in php and have internal css. It is a matter of trial and error and testing.

Here are the thumbnails of the templates
/youinstallation/public/build/img/PDF

**Comment by linknixon** [comment]
Hello,

How about if I would like to change the Currency from USD to UGX because
where I am we use UGX? If this one works I will be glad.

Thank you,

Nixon
